devopstools-jenkins-chefhabitat-kubernetes/README.md

```diff
@@ -0,0 +1,663 @@
+
+9. We can view the output section as shown below.
+
+![alt text](https://github.com/sysgain/azure-quickstart-templates/raw/msoss-p2/devopstools-jenkins-chefhabitat-kubernetes/images/13.png)
+
+## Environment Details:
+
+The ARM template will deploy the following resources on Azure:
+
+
+| S.NO | Nodes                | Installed application            | No of nodes          |Node Purpose                                                                                                       | Ports
+| ---- |-------------         | --------------------             | ------------         |-------------                                                                                                      | -----
+| 1    |Jenkins server        | Jekins                           | 1                    |Install and configure plugins and jobs                                                                             | 8080   
+| 2    |Build Instance        | Chef Habitat                     | 1                    |Creating habitat packages  
+| 3    |ELK Stack             | Elasticsearch, Kibana, Filebeat  | 1                    |Elasticsearch:Contains Index data, Kibana:Segregate logs to visualize as graphs, Filebeat:Forwarding logs to Kibana| 80
+| 4    |Load Balancer         | -                                | 1                    |Directs traffic to Application Nodes                                                                                  |
+| 5    |Azure Storage Account | packer,jenkins,ELK               | 3                    |Packer:To store the Packer VHD’s |
+| 6    |Kubernetes | -             | 1 master, 3 agents                    |To Deploy applications in pods |
+| 7    |Azure Container Registry | -             | 1                  |To store Docker images |
+
+
+## Solution Workflow:
+
+After the template has been successfully deployed, log in to the Jenkins server with its Fully Qualified Domain Name (FQDN) provided in the output section, along with the private SSH key and username.
+
+1. Open PuTTY and enter the Jenkins FQDN under **Session**.
+
+![alt text](https://github.com/sysgain/azure-quickstart-templates/raw/msoss-p2/devopstools-jenkins-chefhabitat-kubernetes/images/14.png)
+
+2. Navigate to **Connection &gt; SSH &gt; Auth**.
+
+![alt text](https://github.com/sysgain/azure-quickstart-templates/raw/msoss-p2/devopstools-jenkins-chefhabitat-kubernetes/images/15.png)
+
+3. Click on the **Browse** section, select SSH private key file which was generated earlier as part of the prerequisites section.
+
+![alt text](https://github.com/sysgain/azure-quickstart-templates/raw/msoss-p2/devopstools-jenkins-chefhabitat-kubernetes/images/16.png)
+
+4. Enter the same username, which was provided while deploying the ARM template.
+
+![alt text](https://github.com/sysgain/azure-quickstart-templates/raw/msoss-p2/devopstools-jenkins-chefhabitat-kubernetes/images/17.png)
+
+
+5. Change to the root user by using the below command:
+
+**sudo -i**
```

Comment:
use code markup for all the commands (here and steps below)

Reply:
updated